docs/modules/deployment/pages/logging.adoc

```diff
@@ -0,0 +1,14 @@
+=== Debugging
```

Comment:
This page seems like a duplicate of data in operations/admin/logging.  Not sure why it's part of the Deployment section.

If this page does stay, suggestions below align the doc with the style guide for structuring files.

```suggestion

[[ga-deployment-debugging]]
=== Debugging
```

Reply:
I agree with Mark. If we keep it, I suggest keeping the first sentence, "When debugging, it may be helpful to lower the minimum severity at which a component will log from the default level of WARN. See xref:operation:admin/logging/introduction.adoc [Logging] for more information and instructions on how to change logging levels.

Reply:
@fooker , this PR has been in my queue for a while. Could we add the cross reference to the logging section, then merge?